webpack.config.js

```diff
@@ -1,5 +1,17 @@
+const webpack = require("webpack");
```

Comment:
Because this one uses `webpack` in its body, to refer to `webpack.optimize.CommonsChunkPlugin`.